Mondee Watch App/View/Play/GameOptionView.swift

```diff
@@ -39,10 +52,47 @@ struct GameOptionView: View {
                 }
                 .frame(maxHeight: .infinity, alignment: .center)
                 .edgesIgnoringSafeArea(.bottom)
+                .alert("게임을 포기하시겠어요?", isPresented: $gameTerminationAlert) {
+                    Button(role: .destructive) {
+                        gameState.giveUpGame()
+                        gameStatus = .fail
+                        withAnimation() {
+                            selection = .game
+                        }
+                    } label: {
+                        Text("오늘 그만")
+                    }
+
+                    Button(role: .cancel) {
+                        
+                    } label: {
+                        Text("취소")
+                    }
+
+                }
+                .alert("정말 청소를 완료하셨어요?", isPresented: $gameEarlySuccessAlert) {
+                    Button {
+                        gameState.successGameEarly()
+                        gameStatus = .success
+                        withAnimation() {
+                            selection = .game
+                        }
```

Comment:
```suggestion
                       selection = .game
```
테스트 결과 withAnimation() 있고 없고 차이가 없는 것 같은데 제거해도 괜찮을까요?